mftf/2.3/test/action-groups.md

```diff
@@ -142,29 +147,35 @@ For example, the following defines the replacement argument `relevantString` usi
 
 The `string` argument type provides a method to pass a single piece of data to the `<actionGroup>`during a test instead of passing an entire entity.
 
-**To explicitly define the argument value**:
+### Define the argument value
 
 ```xml
 <actionGroup stepKey="fillWithStringLiteral" ref="fillExample">
     <argument name="relevantString" value="overrideString"/>
 </actionGroup>
 ```
 
-**To define the argument value using persisted data references**:
+#### Using persisted data references
```

Comment:
It seems to me that these two headings should be at the same level (H3). They should definitely be parallel in construction. I suggest

- Explicitly define the argument value
- Use persisted data references to define the argument value

Reply:
:ok: